### PR TITLE
Add details on using Podman Desktop.

### PR DESCRIPTION
**Pull request by geirst**
@vekterli please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +51,18 @@ Make sure Docker can be executed without sudo for the scripts in this guide to w
...
+
+Install Podman Desktop:
+
+    brew install podman-desktop
+
+Create a new Podman Machine with sufficient resources (Preferences - Resources - Create new ...)
+* CPUs: Minimum 2. Use 8 or more for faster build times.
+* Memory: Minimum 8 GB. 16 GB or more is preferred.
+* Disk size: 128 GB.
+* Machine with root privileges: Enabled
```
Comment (vekterli) [inline README.md:64]: Should we have a [`podman machine init`](https://docs.podman.io/en/stable/markdown/podman-machine-init.1.html) one-liner here with suggested parameters?

Something like
```
podman machine init --cpus=8 --disk-size=128 --memory=16384 --rootful
```
Note: not manually tested 👀

**Review by vekterli — APPROVED** [review]
:+1:

**Comment by geirst** [comment]
@vekterli PTAL

**Review by vekterli — APPROVED** [review]
:+1:

**Pull request merged**